rfc/0021-genericFindQueries.md

```diff
@@ -0,0 +1,162 @@
+# Meta
+
+ - RFC Name: Generic Find Querying
+ - RFC ID: 0021
+ - Start Date: 2016-09-07
+ - Owner: @simonbasle
+ - Current Status: DRAFT
+
+# Summary
+Performing simple selects without writing any N1QL.
```

Comment:
I'd probably phrase this differently.  It's not so much that avoiding N1QL is a goal but rather to keep the simple things simple.  There are lots of situations where just the predicate is needed.